src/components/AwayStatusIndicator.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <span
+        v-if="checkCap()"
+        :user="user"
+        :class="{ 'kiwi-awaystatusindicator-away': isUserAway(),
+                  'kiwi-awaystatusindicator-not-away': !isUserAway(),
+                  'kiwi-awaystatusindicator-self': isUserSelf() }"
+        class="kiwi-awaystatusindicator"
+        @click="toggleSelfAway()"
+    />
+</template>
+
+<script>
+'kiwi public';
+
+export default {
+    props: ['user'],
+    computed: {
+        doesNetworkHaveAwayNotifyCap() {
+            return this.$state.getActiveNetwork().ircClient.network.cap.isEnabled('away-notify');
```

Comment:
The component may not always be in relation to the active network. Eg. multiple network listed in the statebrowser. It will need the network object being passed in

Reply:
Updated, we now check if a network object has been passed in, and if not, we will use the active network.